src/webportal/src/app/home/home/virtual-cluster-statistics.jsx

```diff
@@ -0,0 +1,206 @@
+
+      const resouceUtilization = Math.max(
+        getResouceUtilization(
+          resourcesUsed.GPUs,
+          resourcesTotal.GPUs
+        ),
+        getResouceUtilization(
+          resourcesUsed.memory,
+          resourcesTotal.memory
+        ),
+        getResouceUtilization(
+          resourcesUsed.vCores,
+          resourcesTotal.vCores
+        ),
+      );
+      return (
+        <Stack styles={{root: [{height: 100}]}}>
+          <UtilizationChart percentage={resouceUtilization}/>
+        </Stack>
+      );
+    },
+  },
+  {
+    key: 'detail',
+    minWidth: 250,
+    name: 'Detail',
+    isResizable: true,
+    onRender(vc) {
+      const {resourcesUsed, resourcesTotal} = vc;
+      return (
+        <Stack gap="s1" verticalAlign='center' verticalFill>
+          <StackItem>
+            <ResourceBar
+              name={'Memory'}
+              percentage={getResouceUtilization(
+                resourcesUsed.memory,
+                resourcesTotal.memory
+              )}
+              tailInfo={`${Math.round(resourcesUsed.memory)} / ${Math.round(resourcesTotal.memory)} MB`}
+            />
+          </StackItem>
+          <StackItem>
+            <ResourceBar
+              name={'CPU'}
+              percentage={getResouceUtilization(
+                resourcesUsed.vCores,
+                resourcesTotal.vCores
+              )}
+              tailInfo={`${Math.round(resourcesUsed.vCores)} / ${Math.round(resourcesTotal.vCores)}`}
+            />
+          </StackItem>
+          <StackItem>
+            <ResourceBar
+              name={'GPU'}
+              percentage={getResouceUtilization(
+                resourcesUsed.GPUs,
+                resourcesTotal.GPUs
+              )}
+              tailInfo={`${Math.round(resourcesUsed.GPUs)} / ${Math.round(resourcesTotal.GPUs)}`}
+            />
+          </StackItem>
+        </Stack>
+      );
+    },
+  },
+  {
+    key: 'bonus',
+    minWidth: 60,
+    name: 'Bonus',
+    isResizable: true,
+    onRender(vc) {
+      const bounsEnabled = (vc.maxCapacity > vc.capacity) || vc.capacity === 100;
+      return (
+        <Stack verticalAlign='center' verticalFill>
+          <Text variant='large'>{bounsEnabled ? 'Enabled' : 'Disabled'}</Text>
+        </Stack>
+      );
+    },
+  },
+  {
+    key: 'action',
+    minWidth: 80,
+    name: 'Action',
+    isResizable: true,
+    onRender(vc) {
+      return (
+        <Stack verticalAlign='center' verticalFill>
+          <DefaultButton text={'View jobs'} href={'/job-list.html?vcName=' + vc.name}/>
+        </Stack>
+      );
+    },
+  },
+];
+
+const VirtualClusterStatistics = ({style, virtualClusters}) => {
+  const vcNames = Object.keys(virtualClusters);
+  const {spacing} = getTheme();
+  const vcList = vcNames.map((vcName) => {
+    return {name: vcName, ...virtualClusters[vcName]};
+  });
+
+  return (
+    <Card className={t.ph5} style={{paddingRight: spacing.m, ...style}}>
```

Comment:
I think we need a horizontal scrollbar if the screen is too small (mobile devices)

P.S. For mobile devices I think we could use a sticky header with an expand/collapse button and show all vc items instead of double scrolbars. But this approach needs extra efforts (implmentation, blocked by admin-lte's responsive breakpoint...), I think we could leave it in backlog and discuss it later.

Reply:
Agree, we can create an issue for this

Reply:
#3431